Multisig/UI/Settings/OwnerKeyManagement/LedgerOwnerKey/SelectLedgerDeviceViewController.swift

```diff
@@ -134,6 +134,11 @@ extension SelectLedgerDeviceViewController: BluetoothControllerDelegate {
                                             message: "Please enable Bluetooth in App Settings",
                                             preferredStyle: .alert)
 
+            if let popoverPresentationController = alertVC.popoverPresentationController {
+                //TODO untested
+                popoverPresentationController.sourceView = tableView
```

Comment:
here probably it's not needed because it is an 'alert' style

Reply:
![image](https://user-images.githubusercontent.com/246473/159052451-d00504a1-6098-435f-8257-5d4c9dcb8331.png)

That is xactly, what I was looking for!